articles/iot-hub/iot-hub-devguide-protocols.md

```diff
@@ -43,7 +43,7 @@ Consider the following points when you choose your protocol for device-side comm
 * **Payload size**. MQTT and AMQP are binary protocols, which result in more compact payloads than HTTPS.
 
 > [!WARNING]
-> When using HTTPS, each device should poll for cloud-to-device messages every 25 minutes or more. However, during development, it is acceptable to poll more frequently than every 25 minutes.
```

Comment:
@dcbrown16 Why do you think you would poll less frequently if you're doing development? Wouldn't you want to poll more frequently in that case?

Reply:
The first sentence says to poll "every 25 minutes or more". The second sentence uses "However", implying that it is going to suggest an opposite thing, but then it also says 25 minutes or more. I thought this was trying to say that per the HTTPS guidelines, in production you should make sure to poll every 25 minutes, but in development you don't have to adhere to those guidelines.

Reply:
@dcbrown16 -- I think the wording of the original sentence might be confusing, The idea is that in production you should poll, at most, every 25 minutes. The polling period can be longer if you want. In development, you can relax this standard and poll more frequently if desired. 

Would something like the following make it more clear:

"When using HTTPS, each device should poll for cloud-to-device messages at no more than a maximum frequency of every 25 minutes. You can poll less frequently if desired. During development, however, it is acceptable to poll more frequently than every 25 minutes."

Reply:
Got it, yes "more" was a bit ambiguous in a frequency context. I switched if for @JimacoMS3's  text.

Reply:
@robinsh Can you please have a look here? If you approve, can you go ahead and sign off? I'm pretty sure I don't have those powers :).

Reply:
Hi @dcbrown16 Now that @JimacoMS3 and I have had several discussions about the text, we've come up with this, which we vote for for conciseness. 

When using HTTPS, each device should poll for cloud-to-device messages no more than once every 25 minutes. In development, each device can poll more frequently, if desired.

If this is okay, can you update the article with this, and then sign off on it. (#sign-off in the comment field). It will come to me, and I will sign off on it, and someone will publish it.